### ISSUE TITLE
Perform baddly on validation set

### ISSUE THREAD

**Issue by Choconuts** [snap-research/NeROIC#13]
I use the milkbox dataset to train the model and the training PSNR is about 29.
However, the images synthesis in validation are nearly blank with a little noise, and the PSNR is extremely low. Is it overfitting?

I just run like this for the geometry model: 
``` 
python train.py
--config
configs/milkbox_geometry.yaml
--datadir
data\milkbox_dataset
```

**Comment by zfkuang** [comment]
It might be overfitting. can you send me your log file? 

Also, did you try the other two datasets?

**Comment by Choconuts** [comment]
[version_2.zip](https://github.com/snap-research/NeROIC/files/9331076/version_2.zip)
not yet.

**Comment by zfkuang** [comment]
It looks like your initialization is a bit different from my side, what's your ubuntu version and did you use the conda command in the guide to install all the packages? 

![Screen Shot 2022-08-13 at 1 44 57 AM](https://user-images.githubusercontent.com/20102505/184476343-c5fec8f6-d436-4f61-93c4-171ba5bdb7ac.png)
My initialization

![Screen Shot 2022-08-13 at 1 43 15 AM](https://user-images.githubusercontent.com/20102505/184476302-b3ba14f8-37e4-4a97-961b-9d151c44d0a3.png)
Yours

**Comment by Choconuts** [comment]
I use windows. Does it matter so much? I mean initialization, since it can converge on training-set easily after all. 
How can I correct the initialization?

**Comment by zfkuang** [comment]
It might be, but there're some other things that looks odd: for example your camera loss is extremely high, indicating that the system didn't find a good camera pose either. So my guess is that there're some nuances between the packages on different platforms causing all the trouble. I would suggest using ubuntu, because all experiments are conducted on it. I can also test the model on windows later to see how to fix that.

**Comment by Choconuts** [comment]
OK, thank you.
